src/MUIDataTable.js

```diff
@@ -19,6 +19,8 @@ import DefaultTableToolbarSelect from './components/TableToolbarSelect';
 import MuiTooltip from '@material-ui/core/Tooltip';
```

Comment:
This component is getting very complicated. (Because it's the **main**...) 😭

What do you think about this?

Is there a better way?

Reply:
It is, though it'll inherently be the most complex piece. I think I'll try and move some of what I wrote out of main and only keep what's essential there.